DataFormats/Reconstruction/include/ReconstructionDataFormats/Vertex.h

```diff
@@ -40,7 +40,7 @@ class VertexBase
                         kCovYZ,
                         kCovZZ };
   static constexpr int kNCov = 6;
-  GPUdDefault() VertexBase() = default;
+  GPUhd() VertexBase() = default;
```

Comment:
@davidrohr  Whereas here I'm not 100% sure.

Reply:
Not sure if this will pass the fullCI, let's see. Again, this means you are creating a VertexBase() in the host part during GPU compilation. Is that needed?

Reply:
E.g.: when you want to pull back vertices from GPU to plug into tracker. I just had to add this for that reason.

Reply:
OK, so you have also host code that works on the data in the `.cu` / `.hip` files. This is fine with me as long as the FullCI is green.